packages/form-js-viewer/src/render/components/form-fields/parts/TemplatedInputAdorner.js

```diff
@@ -0,0 +1,17 @@
+import InputAdorner from './InputAdorner';
+
+import { useSingleLineTemplateEvaluation } from '../../../hooks';
+
+export default function TemplatedInputAdorner(props) {
+
+  const {
+    pre,
+    post
+  } = props;
+
+  const evaluatedPre = useSingleLineTemplateEvaluation(pre, { debug: true });
```

Comment:
is the `debug: true` expected to be here still?

Reply:
Yeah it is a property to make `feelers` not throwing errors on invalid feel expressions. We could consider making this the default in the `useSingleLineTemplateEvaluation` hook though.

Wdyt?

Reply:
Yeah it is a property to make `feelers` not throwing errors on invalid feel expressions. We could consider making this the default in the `useSingleLineTemplateEvaluation` hook though.

Wdyt?

Reply:
I think it's ok to leave it as is, I was just wondering if this was something that was meant for use while testing only